jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -430,7 +433,7 @@ case "`uname`" in
 CYGWIN*) JETTY_START="`cygpath -w $JETTY_START`";;
 esac
 
-RUN_ARGS=(${JAVA_OPTIONS[@]} -jar "$JETTY_START" ${JETTY_ARGS[*]})
+RUN_ARGS=$(echo $JAVA_OPTIONS ; "$JAVA" -jar "$JETTY_START" --dry-run=opts,path,main,args ${JETTY_ARGS[*]} | tail -1 )
```

Comment:
I don't like the `tail -1` as in "why do we take only the last line?" "surely if there are multiple lines we don't want to take only the last one and discard the previous ones".
I'd rather fail to start if people puts newlines into `JAVA_OPTIONS`.

Reply:
We have no control over what JAVA_OPTIONS users set. We have no control over what output those options generate or indeed if the JVM generates any other output.
If fact I think we should do more, something like:
```
 java --show-version -jar ../jetty-home/start.jar --dry-run=opts,path,main,args __jetty__=__args__ 2>/dev/null | grep __jetty__=__args__ | sed 's/ __jetty__=__args__ / /'
```

Reply:
I think this is simpler!

Reply:
> We have no control over what JAVA_OPTIONS users set.

That's exactly why we should do exactly nothing to try to format it. We know our `--dry-run` generates a single line, and if they set JAVA_OPTIONS right, then there is nothing else to do, no `tail`, no `sed`, etc.

We don't do `tail -1` on the other variables in this file that can be set by users, so why do it here?
Am I missing something?